shared/test/test_assets.rb

```diff
@@ -56,7 +56,17 @@ def test_assets
     delete(@assets, channel_id, 'filename.jpg')
     assert @assets.last_response.successful?
 
-    # invalid files are not uploaded
+    # file extension case insensitivity
+    put(@assets, channel_id, 'filename.JPG', 'stub-contents', 'application/jpeg')
+    assert @assets.last_response.successful?
+    get(@assets, channel_id, 'filename.JPG')
+    assert @assets.last_response.successful?
+    get(@assets, channel_id, 'filename.jpg')
+    assert @assets.last_response.not_found?
```

Comment:
Hi @joshlory , Sarah's answer in the pivotal item is that either way is fine. Any further concerns?